regen-network/regen-web
# feat: APP-330 disable "edit profile" and "settings" user menu items if using …

## Description

https://regennetwork.atlassian.net/browse/APP-330

---

### Author Checklist

I have...


### How to test

1. Login with Wallet Connect or Keplr Mobile Web
2. Verify that the user menu has the Edit Profile and Settings disabled and when you hover over a tooltip opens up with the copy: “This is not yet supported by Wallet Connect and Keplr mobile. Please log in using Keplr browser extension on desktop"
3. The behaviour should be the same on mobile but when touching the menu items.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
